listenbrainz/listenstore/tests/test_timescalelistenstore.py

```diff
@@ -426,30 +426,39 @@ def test_listen_counts_in_cache(self):
         self.assertEqual(count + 1, int(cache.get(user_key, decode=False) or 0))
 
     def test_delete_listens(self):
-        self._create_test_data(self.testuser_name)
-        listens, min_ts, max_ts = self.logstore.fetch_listens(user_name=self.testuser_name, to_ts=1400000300)
+        uid = random.randint(2000, 1 << 31)
```

Comment:
I think it would make sense to fetch the cache values after deleting the listens and making sure they are what we expect.

Reply:
Good idea. Done.